libMBIN/Source/NMS/GameComponents/GcAtlasActiveFeaturedBaseHeader.cs

```diff
@@ -1,3 +1,6 @@
+using System;
```

Comment:
this isn't actually needed I don't think...

Reply:
It is not needed specifically for this one, but it is needed for a few structure files (those Gc using [Flags] and many Tk ones).  So my automated script includes it by default.  It does not cause any problem being present.  It does not add much processing time to any build.  I would keep it.